src/plugins/syscalls/win.h

```diff
@@ -3764,7 +3781,14 @@ SYSCALL(NtUserManageGestureHandlerWindow, NTSTATUS, 0);
 SYSCALL(NtUserMapPointsByVisualIdentifier, NTSTATUS, 0);
 SYSCALL(NtUserMapVirtualKeyEx, NTSTATUS, 0);
 SYSCALL(NtUserMenuItemFromPoint, NTSTATUS, 0);
-SYSCALL(NtUserMessageCall, NTSTATUS, 0);
+SYSCALL(NtUserMessageCall, NTSTATUS, 7,
```

Comment:
Please move this above to be placed with the other defined syscalls

Reply:
@tklengyel thanks for note. Done!